corehq/apps/app_manager/templates/app_manager/module_view.html

```diff
@@ -77,6 +77,7 @@
   {% initial_page_data 'sessionid' request.COOKIES.sessionid %}
   {% initial_page_data 'default_language' app.default_language %}
   {% initial_page_data 'current_language' lang %}
+  {% initial_page_data 'app_analytics' app_analytics %}
```

Comment:
Feature flags are available to javascript - grep around for `hqwebapp/js/toggles` for examples.